### PR TITLE
Add Open Previous Archive and Open Next Archive

### LINKED ISSUE #9: Keyboard shortcuts to open the previous/next archive
More specifically, this would be the previous/next archive in the directory of the currently opened archive

### PR DESCRIPTION
**Pull request by the-blank-x**
Fixes #9

### CODE REVIEW

File: src/mainwindow.h [hunk 1]
```diff
@@ -53,6 +53,7 @@ class MainWindow : public KXmlGuiWindow
     void setupActions();
     void openMangaFolder();
     void openMangaArchive();
+    void openCloseArchive(bool next);
```
Comment (the-blank-x) [inline src/mainwindow.h:56]: I don't really like the name `openCloseArchive`, but I can't really think of a good alternative. `openNextArchive` would be too ambiguous (next as in the next one in the list, or next as in close to?)

Comment (g-fb) [reply src/mainwindow.h:56]: Why didn't you add 2 functions openNextArchive and openPreviousArchive? 

You don't have to change it now, but I might do it later.

Comment (the-blank-x) [reply src/mainwindow.h:56]: Having two functions with the same function body (minus an offset) feels a bit overkill

**Comment by the-blank-x** [comment]
Oh, okay. https://github.com/g-fb/mangareader#status does say "If someone else wants to add a feature they are welcome to submit a PR, but before doing that open an issue to check if the feature is gonna be accepted." (though it probably was expected to there be a wait until a response)

**Pull request merged**